packages/web3-providers-base/src/index.ts

```diff
@@ -0,0 +1,28 @@
+// TODO Fix eslint issues
+// eslint-disable-next-line import/no-unresolved, import/extensions
+import { ProviderOptions } from '../types';
+
+export default class Web3ProviderBase {
+  providerString = ''
+
+  protectProvider = false
+
+  connected = false
+
+  supportsSubscriptions = false
```

Comment:
Both properties aren't being used, are they needed right now?

Reply:
Right now I've only written the HTTP package, so no, but they will be needed for WS and maybe IPC

Reply:
So maybe let those PR's introduce them, since you haven't written the ws you're making an assumption on the architecture. just a nit

Reply:
You're right, they've been removed :)